app/client/components/creditTransfer/bulkTransferList.tsx

```diff
@@ -94,6 +94,8 @@ const columns: ColumnsType<BulkTransfer> = [
         return <Tag color="#9bdf56">Approved</Tag>;
       } else if (record.state === "PENDING") {
         return <Tag color="#e2a963">Pending</Tag>;
+      } else if (record.state === "PARTIAL") {
+        return <Tag color="#ecec28">Partial</Tag>;
```

Comment:
This colour is a bit hard to read - can we change it?

https://ant.design/components/tag/

Reply:
Went with the `gold` border colour, looks pretty good! 
![image](https://user-images.githubusercontent.com/2199643/115081147-2f62df00-9eda-11eb-9470-102759d20291.png)

Reply:
Nice!
Nit: do you think we make the gold the same as the `Vendor` on accounts page? -> `#e2a963`

Reply:
Just realized that would make it the same as `PENDING` status. I think leave as is for now then